pkg/testing/tools/artifacts_api.go

```diff
@@ -275,8 +275,9 @@ func GetLatestSnapshotVersion(ctx context.Context, log logger, aac *ArtifactAPIC
 		return nil, ErrSnapshotVersionsEmpty
 	}
 
-	// normally the output of the versions returned by artifact API is already sorted in ascending order,
-	// if we want to sort in descending order we could use
+	// normally the output of the versions returned by artifact API is already
+	// sorted in ascending order.If we want to sort in descending order we need
+	// to pass a sort.Reverse to sort.Sort.
```

Comment:
After discussing the meaning of this comment maybe we can clarify the wording a bit like
```suggestion
	// normally the output of the versions returned by artifact API is already
	// sorted in ascending order. Since we want to extract the most recent version,
	// we sort in descending order by passing a sort.Reverse operator to sort.Sort.
```